Fix transparent background in the favicon

discodavey opened the pull request:
Have not removed dependencies as yet in case we need to keep gulp-svg2png

discodavey commented:
Border has now gone omn Chrome browsers. Have checked on iPad and iPhone as well

thewilkybarkid commented:
Is the problem `gulp-svg2png` and/or the SVG (as that's been changed too)?

> Have not removed dependencies as yet in case we need to keep gulp-svg2png

Let's remove it. Easy enough to add back in, and best not to have things kept around unused.

thewilkybarkid commented:
The conversion was added in https://github.com/elifesciences/journal/pull/412. Might not be a problem anymore due to the use of containers.

discodavey commented:
I have removed each of the things that make up the gulp favicons build and this is the only thing that is causing the border to appear around the circle. I changed the variables on imagemin as well as favicons and did not make any difference. 

I believe that svg2png is the issue. It is still producing the icons for png's.

Can you see anything that could cause issues? @thewilkybarkid ?

thewilkybarkid commented:
Taking a look.

thewilkybarkid commented:
The problem appears to be https://github.com/elifesciences/journal/pull/1244/files#diff-088bff340207d12e84b750013697f982R6, removing it seems to remove the problem with no adverse effects.

Definitely gives a bit more weight to https://github.com/elifesciences/pattern-library/pull/970#issuecomment-595298875, having redundant structure can cause problems elsewhere.

thewilkybarkid commented:
Needs renaming though.

discodavey commented:
What needs renaming?

thewilkybarkid commented:
The PR (and the resulting merge commit).

discodavey commented:
Changed the title

thewilkybarkid commented on assets/images/favicon.svg:
```diff
@@ -3,7 +3,6 @@
     <title>journal-symbol-no-hoop</title>
     <desc>Created with Sketch.</desc>
-    <g id="journal-symbol-no-hoop" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
         <rect id="Rectangle" fill="#FFFFFF" x="0" y="0" width="500" height="500"></rect>
```
https://github.com/elifesciences/journal/pull/1249/commits/acecfaa918d7ea73f76328153671b069bc9f04c7 needs undoing too doesn't it?

thewilkybarkid commented:
> Removing code with stroke-width

Describes the what rather than the why.

> Chrome browsers favicons

Not exactly right, it's PNG and ICO favicons (so isn't just limited to Chrome/Chromium).

Suggestion: 'Fix transparent background in the favicon'.

discodavey commented:
Are we happy with this? Can I squash and merge? @thewilkybarkid

discodavey commented:
Do not see the need to open another PR as this was opened for fixing the issue and gulp-svg2png is one of the things that is causing the current issue without looking into the svg itself.

Just think It's just adding complication by having another PR to go through.
